impress-org/givewp
# "Donor Profiles" are now renamed "Donor Dashboard"

Resolves #5682 

## Description

This PR renames references to Donor Profiles as Donor Dashboards. This PR should change most references in the codebase, and all visual references to Donor Dashboards. Notably: The PR does not update the domain namespace, which is still DonorProfiles. This will be addressed in a future PR, when addons which rely on the domain namespace will also be updated.

## Affects

This PR affects many, many facets of the DonorProfiles domain. Primarily, it is meant to ensure that any visual representation of the donor profile feature is now referred to as the Donor Dashboard.

## Visuals

N/A

## Testing Instructions

1. Setup a new Donor Dashboard
2. Test various functionality to ensure that nothing is newly broken
3. Check copy for ferences to Donor Profiles (including contextual docs, generated page title, block name and shortcode).

## Pre-review Checklist
